src/objects/embed/embed.twig

```diff
@@ -5,7 +5,7 @@
 <div class="o-embed{% if class %} {{class}}{% endif %}"
   {% if style %}
     style="
-      {%- for rule in style|split(';') if rule|length -%}
+      {%- for rule in style|split(';')|filter(rule => rule|length) -%}
```

Comment:
Yay, the ES6-style Twig filters work now! (They didn't in Twig.js)

Reply:
Yeah, I love this syntax! 🎉